modules/nf-core/glimpse/concordance/main.nf

```diff
@@ -0,0 +1,46 @@
+process GLIMPSE_CONCORDANCE {
+    tag "$meta.id"
+    label 'process_low'
+
+    conda "bioconda::glimpse-bio=1.1.1"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/glimpse-bio:1.1.1--hce55b13_1':
+        'quay.io/biocontainers/glimpse-bio:1.1.1--hce55b13_1' }"
+
+    input:
+    tuple val(meta), val(region), path(freq), path(truth), path(estimate)
+    val(min_prob)
+    val(min_dp)
+    val(bins)
```

Comment:
Are these mandatory arguments?

Reply:
Yes there are.

Reply:
Perfect! We're having a meeting in the coming weeks about this but for now it's fine as long as they are mandatory :)